lib/iamport.rb

```diff
@@ -48,92 +50,103 @@ def payments(options = {})
 
       uri = "payments/status/#{status}?page=#{page}"
 
-      _get(uri)
+      pay_get(uri)
     end
 
     # Find payment information using merchant uid
     # https://api.iamport.kr/#!/payments/getPaymentByMerchantUid
     def find(merchant_uid)
       uri = "payments/find/#{merchant_uid}"
 
-      _get(uri)
+      pay_get(uri)
     end
 
     # Prepare payment validation
     # https://api.iamport.kr/#!/payments/prepare
     def prepare(body)
       uri = "payments/prepare"
 
-      _post(uri, body)
+      pay_post(uri, body)
     end
 
     # Get prepared payment information by merchant_uid
     # https://api.iamport.kr/#!/payments/prepare/:merchant_uid
     def prepared(merchant_uid)
       uri = "payments/prepare/#{merchant_uid}"
 
-      _get(uri)
+      pay_get(uri)
     end
 
     # Canceled payments
     # https://api.iamport.kr/#!/payments/cancelPayment
     def cancel(body)
       uri = "payments/cancel"
 
-      _post(uri, body)
+      pay_post(uri, body)
     end
 
-    # Get a billing key by customer_uid
-    # GET https://api.iamport.kr/#!/subscribe/customers/:customer_uid
-    def find_subscribe_customer(customer_uid)
```

Comment:
How do you think about let old method still remain and `warn` deprecated method message to give some times for the lower version compatibility?

Reply:
@jhsbeat I updated - https://github.com/iamport/iamport-rest-client-ruby/pull/20/files#diff-f78534a315fc8994b2f7f9dffaa4d844R96